charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml

```diff
@@ -274,6 +277,30 @@ data:
       - target_label: instance
         replacement: kube-state-metrics
       metric_relabel_configs:
+      # We want to drop all metrics that
+      # do not belong to the shoot
+
+      # Add a temp label to all ksm metrics
+      - target_label: __tmp_keep
```

Comment:
Although I kind of understand what's happening here, I would like to think about an alternative to make it easier to understand.
What about adding a second scrape config for the seed ksm, which keeps only metrics for the current Shoot namespace, (and rewrites the job name to `kube-state-metrics` at the end, if needed)?

Reply:
I discussed with @istvanballok and we have a few ideas on how we can simplify the config. Unfortunately it is a bit difficult since it is not possible to do negations with this implementation of the regex engine.

Reply:
I assume, having a separate scrape config for the Seed ksm is not possible?

Reply:
it is possible but there are alerts and dashboards that reference the the job `kube-state-metrics`. This is why I wanted to keep it as one job. If you prefer I can change it to 2 jobs

Reply:
Hmm, ok. I would prefer to split it up into 2 jobs. But if that's too much effort, we can leave it like this (although, I would say that 2 jobs are the cleaner way).
You can choose :)